src/prost/src/lib.rs

```diff
@@ -221,13 +221,21 @@ impl stream_plan::MaterializeNode {
 
 // Encapsulating the use of parallelism.
 impl common::WorkerNode {
-    pub fn parallelism(&self) -> usize {
+    pub fn compute_parallelism(&self) -> usize {
```

Comment:
Does this new name look like "computing" the parallelism when it's called? What about `cn_parallelism` or `cnode_parallelism`?